packages/uploadthing/src/internal/types.ts

```
@@ -157,32 +140,45 @@
     _errorFn: UploadErrorFn;
     _output: UnsetMarker;
   }>;
+  onUploadComplete: <TOutput extends JsonObject | void>(
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)